ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/schema/ExecutionPayloadV1.java

```diff
@@ -69,22 +69,25 @@ public ExecutionPayloadV1(
   }
 
   public ExecutionPayload asInternalExecutionPayload(
-      ExecutionPayloadSchema executionPayloadSchema) {
-    return executionPayloadSchema.create(
-        parentHash,
-        feeRecipient,
-        stateRoot,
-        receiptsRoot,
-        logsBloom,
-        prevRandao,
-        blockNumber,
-        gasLimit,
-        gasUsed,
-        timestamp,
-        extraData,
-        baseFeePerGas,
-        blockHash,
-        transactions);
+      ExecutionPayloadSchema<?> executionPayloadSchema) {
+    return executionPayloadSchema
+        .toVersionBellatrix()
```

Comment:
Why only Bellatrix, will it be some `ExecutionPayloadV2` for Capella?

Reply:
Yeah, this is another place where things need to be changed. Ideally would like to tackle this in separate PR. Probably when we make the engine API changes.